plugins/altos/fu-altos-firmware.c

```diff
@@ -17,6 +17,12 @@ struct _FuAltosFirmware {
 
 G_DEFINE_TYPE (FuAltosFirmware, fu_altos_firmware, FU_TYPE_FIRMWARE)
 
+static void
+fu_altos_firmware_to_string (FuFirmware *firmware, GString *str)
+{
+	g_string_append (str, "FuAltosFirmware:\n");
```

Comment:
Couldn't this be automatically stubbed with the type of plugin by `FuFirmware`?

Reply:
Doh, yes of course. Will fix.

Reply:
Okay, much less new code now, thanks.